static/app/components/button.tsx

```diff
@@ -82,16 +89,11 @@ function BaseButton({
     return disabled ? undefined : prop;
   }
 
-  // For `aria-label`
-  const screenReaderLabel =
-    ariaLabel || (typeof children === 'string' ? children : undefined);
```

Comment:
No need to set aria-label explicitly. Scree readers natively understand this scenario.